src/amo/components/Home/styles.scss

```diff
@@ -73,6 +87,10 @@
   padding: 0;
   width: 100%;
 
+  @include respond-to(extraSmall) {
```

Comment:
We don't have a breakpoint named `extraSmall` defined in https://github.com/mozilla/addons-frontend/blob/851f05b08ece38bc475f9be42d348f06eb15fa7b/src/core/css/inc/vars.scss#L59

Do you still need to add this rule? It looks like things are working without it.

Reply:
Ya, this breakpoint is used to generate this screen for viewports of size less than 349px as suggested by @pwalm. 
https://user-images.githubusercontent.com/6797635/36923791-3948abc2-1e21-11e8-83c7-48fd86d5a92b.png

Reply:
hi @hritvi : ) since we do mobile first approach you should be able to set the mobile styles in the initial rules and then you can override those styles for example at the medium (500px)/higher breakpoint (see already defined breakpoints [here](https://github.com/mozilla/addons-frontend/blob/851f05b08ece38bc475f9be42d348f06eb15fa7b/src/core/css/inc/vars.scss#L59))

```css 
.Home-SubjectShelf-list {
   flex-flow: column wrap; // mobile
   ...other styles

   @include respond-to(medium) {
      flex-flow: row wrap // overrides mobile
      ...override other mobile styles or set new ones for this bp here
    }
}
```